search-commons/src/main/java/no/unit/nva/search2/ticket/TicketSearchQuery.java

```diff
@@ -273,5 +274,12 @@ protected void setValue(String key, String value) {
         protected boolean isKeyValid(String keyName) {
             return TicketParameter.keyFromString(keyName) != TicketParameter.INVALID;
         }
+
+        private String expandMissing(String value) {
```

Comment:
Just a suggestion to make the method more self explanatory. Don't have to change if you don't want. 

```suggestion
        private String expandIfMissingRange(String value) {
```